app/views/layouts/course.html.slim

```diff
@@ -1,13 +1,12 @@
 - layout = controller.parent_layout(of_layout: 'course') || controller.current_layout
 = render within_layout: layout do
   = render_breadcrumbs
-  div.row
+  div.row.course-layout
```

Comment:
So you applied the diff I commented with? The CSS would need to be updated also.